ckanext/search/filters.py

```diff
@@ -115,66 +128,80 @@ def _combine_filter_operator_members(
 
 
 def _process_filter_operator_members(
-    value: Any, parent_operator: str
+    value: Any, parent_operator: str, search_schema: dict
 ) -> tuple[Optional[List[FilterOp]], Optional[List[str]]]:
 
     if not isinstance(value, list):
         return None, [f"Filter operations must contain lists of filters: {value}"]
 
     out: List[FilterOp] = []
     errors = []
+
     for item in value:
+        item_errors = []
         if not isinstance(item, dict):
-            errors.append(f"Filter operation members must be dictionaries: {item}")
+            item_errors.append(f"Filter operation members must be dictionaries: {item}")
```

Comment:
Since the error is about the type maybe the repr is better?
```suggestion
            item_errors.append(f"Filter operation members must be dictionaries: {item!r}")
```
or even `{json.dumps(item)}`  so it looks right for API users?